src/test/java/core/basesyntax/model/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.model;
+
+import core.basesyntax.service.InvalidUserDataException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_ValidUser_SuccessfulRegistration() throws InvalidUserDataException {
+        User user = new User();
+        user.setLogin("testuser");
+        user.setPassword("strongpassword");
+        user.setAge(20);
```

Comment:
Please move this logic to BeforeEach annotated method.
You will have standard valid user before each test.
So you can change only one field that you are going to test (only password or login etc.)